docs/content/users/extend/custom-compose-files.md

```diff
@@ -1,22 +1,22 @@
+Much of DDEV’s customization ability and extensibility comes from leveraging features and functionality provided by [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/overview/). Some working knowledge of these tools is required in order to customize or extend the environment DDEV provides.
 
-There are [many examples of custom Docker compose files](https://github.com/drud/ddev-contrib#additional-services-added-via-docker-composeserviceyaml) available on [ddev-contrib](https://github.com/drud/ddev-contrib).
+There are [many examples of custom docker-compose files](https://github.com/drud/ddev-contrib#additional-services-added-via-docker-composeserviceyaml) available on [ddev-contrib](https://github.com/drud/ddev-contrib).
 
 ## Background
 
-Under the hood, ddev uses a private copy of docker-compose to define and run the multiple containers that make up the local environment for a project. docker-compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and ddev is designed to leverage this ability.
+Under the hood, DDEV uses a private copy of docker-compose to define and run the multiple containers that make up the local environment for a project. docker-compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and DDEV is designed to leverage this ability.
 
-To add custom configuration or additional services to your project, create docker-compose files in the `.ddev` directory for your project. ddev will process any files using the `docker-compose.[servicename].yaml` naming convention and include them in executing docker-compose functionality. Optionally you can also create a `docker-compose.override.yaml` to override any configurations from the main `.ddev/.ddev-docker-compose-base.yaml` or any additional docker-compose files added to your project.
+To add custom configuration or additional services to your project, create docker-compose files in the its `.ddev` directory. DDEV will process any files with the `docker-compose.[servicename].yaml` naming convention and include them in executing docker-compose functionality. You can optionally create a `docker-compose.override.yaml` to override any configurations from the main `.ddev/.ddev-docker-compose-base.yaml` or any additional docker-compose files added to your project.
 
-!!!note Don't use or edit the `.ddev-docker-compose-base.yaml` or `.ddev-docker-compose-full.yaml`
+!!!warning "Don’t modify `.ddev-docker-compose-base.yaml` or `.ddev-docker-compose-full.yaml`!"
```

Comment:
```suggestion
!!!warning "You will never modify the generated `.ddev-docker-compose-base.yaml` or `.ddev-docker-compose-full.yaml`!"
```

Reply:
Do you feel strongly about changing this tip’s *heading*, @rfay? I think shorter is better since it has supporting text, and “you will never” has a bit of a harsh, scolding-parent vibe where “don’t” is a more clear, concise direction. I can also _try_ to edit those files (defying “never”), it’ll just be a pointless and frustrating experience.

If anything, I’d work “generated” into the supporting text in case “overwritten” is too vague.

Reply:
Whatever is fine. Not sure why "You will never" would be better than "Don't"

Reply:
I guess just that some people go in there and do it. Maybe it happened that day :)